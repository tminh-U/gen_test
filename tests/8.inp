9
15 18 19 17 11 18 17 17 8 
13
3 6
2 8
3 6
8 9
8 8
5 7
4 8
9 9
2 4
2 2
1 9
5 8
5 8
