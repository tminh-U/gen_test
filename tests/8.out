7
19
7
3
2
6
15
2
5
2
34
10
10
